v2.2/follower-reads.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Follower Reads
+summary: To reduce latency for `SELECT` operations, you can choose to have the closest node serve the request using the "Follower Reads" feature.
+toc: true
+---
+
+<span class="version-tag">New in v2.2:</span> To reduce latency for `SELECT` operations, you can choose to have the closest node serve the request using the "Follower Reads" feature. The trade-off here is that the response might not have the most recent version of the data, though the data will be consistent at the point in time the read is served from.
+
+{{site.data.alerts.callout_danger}}
+The Follower Read feature is only available to [enterprise](https://www.cockroachlabs.com/product/cockroachdb/) users.
+{{site.data.alerts.end}}
+
+## Settings
+
+### Enable/disable Follower Reads
+
+Use [`SET CLUSTER SETTING`](set-cluster-setting.html) to set `kv.range_split.by_load_enabled` to:
+
+- `false` to disable follower reads _(default)_
+- `true` to enable follower reads 
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SET CLUSTER SETTING kv.closed_timestamp.follower_reads_enabled = true;
+~~~
+
+#### When to use Follower Reads
+
+Follower Reads return consistent historical reads; currently a minimum of 48 seconds in the past, though we are actively working on reducing that number.
```

Comment:
I feel like somewhere in this document it's worth explaining what a follower read is. 

I might also use a quite comment about how a follower read differs from a leaseholder (maybe call it transactional) read.
```
Unlike transactional reads which must be fulfilled by the current leaseholder for a range, follower reads can be served safely by any replica. In order to utilize safe and consistent follower reads users must be willing to sacrifice some amount of data freshness; follower reads only apply to historical read queries (using AS OF SYSTEM TIME) which are sufficiently in the past. Currently that bar is calculated to be 48 seconds though there is active work to lower that number. A function described below can be provided to AS OF SYSTEM TIME to request a timestamp which is as close to the present as possible yet safe for follower reads.
```